sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/json/JacksonGeneratorSuite.scala

```diff
@@ -39,6 +39,18 @@ class JacksonGeneratorSuite extends SparkFunSuite {
     assert(writer.toString === """{"a":1}""")
   }
 
+  test("initial with StructType and write out an empty row with allowStructIncludeNull=true") {
```

Comment:
This looks like a test case for bug. We need a JIRA issue ID prefix for the test case name.

Reply:
ok